stats/lib/stats-fetchers.js

```diff
@@ -270,3 +270,48 @@ export const fetchDailyRetrievalResultCodes = async (pgPools, filter) => {
   const stats = Object.entries(days).map(([day, rates]) => ({ day, rates }))
   return stats
 }
+
+/**
+ * Fetches daily global retrieval time statistics
+ * @param {import('@filecoin-station/spark-stats-db').PgPools} pgPools
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchDailyRetrievalTimes = async (pgPools, filter) => {
+  const { rows } = await pgPools.evaluate.query(`
+    SELECT
+      day::text,
+      percentile_cont(0.5) WITHIN GROUP (ORDER BY time_to_first_byte_p50) AS ttfb_p50
```

Comment:
Having variables `time_to_first_byte_p50` and `ttfb_p50` is confusing, as the names don't tell us how they differ. Can you think of a more semantic name for the computed column?

Reply:
There are two aspects to consider:

1. What name do we want to use in the SQL query?
2. What property name do we want to use in the response body?

Also, note that a p50 value calculated from a list of p50 values is not a true global p50 value; it's just an approximation we decided to use to keep this initial implementation simple.

In that light, I propose to use the name `ttfb` in the response.

```json
[
  { "day": "2025-01-08", "ttfb": 850 }
  { "day": "2025-01-09", "ttfb": 930 }
]
```

I think this name works well for the SQL query, too, but I don't mind using a different name there.

Reply:
One more idea to consider - should we encode the unit (milliseconds) in the property name?

```json
[
  { "day": "2025-01-08", "ttfb_ms": 850 }
  { "day": "2025-01-09", "ttfb_ms": 930 }
]
```

Reply:
`"ttfb_ms"` lgtm!